### PR TITLE
DOT-7352 > COD theme > Redesign the main page part 2

### PR DESCRIPTION
**Pull request by adildev101**
## JIRA Ticket

[DOT-7352](https://youcanshop.atlassian.net/browse/DOT-7352)

## QA Steps

-  [ ] `pnpm i`
-  [ ] `pnpm dev`

## Note

Leave empty when you have nothing to say

[DOT-7352]: https://youcanshop.atlassian.net/browse/DOT-7352?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: assets/reviews-count.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    const reviews = await youcanjs.product.fetchReviews(productId).data();
+    const reviewCount = reviews.length;
+
+    reviewsContainer.style.display = 'block';
+    reviewsWrapper.innerHTML = `
+      <div class='rating-stars'>
+        <div class="yc-reviews-stars" style="--rating: ${calculateAverageRating(reviews)};" aria-label="Rating of this product is ${calculateAverageRating(reviews)} out of 5"></div>
+      </div>
+      <div class='review-count'>
+        (${reviewCount} تقييمات)
```
Comment (jamyoucn00) [inline assets/reviews-count.js:20]: the word `تقييمات` should not be static, use the translation filter or let the user add it (up to you 🖌️:P)

File: assets/reviews-count.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+function calculateAverageRating(reviews) {
+  if (!reviews || !reviews.length) {
+    return 0;
+  }
+
+  const totalRating = reviews.reduce((sum, review) => {
+    return sum + review.ratings;
+  }, 0);
+
+  return totalRating / reviews.length;
```
Comment (jamyoucn00) [inline assets/reviews-count.js:41]: if the reviews.length was equal to 0, what will happen?

File: sections/product-column.liquid [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+        "label": "Description",
+        "limit": 1,
+        "type": "product_description"
+      }
+    ]
+  }
+{%- endschema -%}
+
+
+{{ 'product.js' | asset_url | script_tag_deferred }}
```
Comment (jamyoucn00) [inline sections/product-column.liquid:134]: can you please add an extra line here

File: styles/main.scss [hunk 4]
```diff
@@ -474,3 +474,37 @@ input:not([type='radio']),
...
+  &.top_left {
+    top: 8px;
+    left: 8px;
+  }
+
+  &.bottom_left {
+    bottom: 8px;
+    left: 8px;
+  }
+}
```
Comment (jamyoucn00) [inline styles/main.scss:510]: extra line

File: assets/reviews-count.js (hunk 2 above)
Comment (adildev101) [reply assets/reviews-count.js:41]: will never have 0 since we can't send 0 in reviews if their is no reviews will not appear

**Review by jamyoucn00 — APPROVED** [review]

**Pull request merged**